tests/system/components/authHeader.js

```diff
@@ -106,4 +107,35 @@ module.exports = {
         .assert.attributeContains('section > div > div > div > div > div:nth-child(4) > ul > li:nth-child(2) > a', "href", "help")
         .end();
     }
```

Comment:
Because you are adding more tests, you need to insert a comma here, or you will get js error